public_dropin_environments/python311_genai/requirements.txt

```diff
@@ -19,3 +19,4 @@ pydantic==2.2.1
 pydantic-settings==2.0.3
 aiofiles==23.1.0
 aioboto3==12.1.0
+loguru==0.7.2
```

Comment:
I'm wondering if we can move the logging in https://github.com/datarobot/buzok/pull/863/files and avoid the need for it here. I'm not sure where that log shows up when the custom model is executed.